### PR TITLE
[Accessibility] [Keyboard Navigation] Enable left panels to focus on the first interactive element

### PR DESCRIPTION
**Pull request by luislera**
### Describe the issue

If users select Resources control and pane gets appear but keyboard focus indicator remain on the same control and does not land on first interactive control on Resources pane, then it would lengthy process to users to navigate on Resources pane. The user needs to navigate on all control to move on the Resources pane.

**Actual behavior:**

After selecting the Resource control pane gets appears but the keyboard focus indicator remains on the same control, it does not land on the first interactive control on the Resource pane.

**Expected behavior:**

After selecting the Resource control pane gets appears, the keyboard focus indicator should be land in the first interactive control on the Resource pane, it should not remain on the same control.

**Repro Steps:**

1. Open Bot Framework Emulator installed on the system.
2. Bot Framework Emulator Application with Welcome screen opens.
3. Navigate on the welcome screen and select Create a New BOT configuration link.
4. New BOT Configuration dialog opens.
5. Navigate on the dialog and enter inputs in edit fields then select the save and connect button.
6. Navigate to the Resources icon on the left pane and select it.
7. Resources Pane opens, navigate on the pane.
8. Verify that focus land on the first interactive control on the resource pane or not.

### Changes included in the PR

- Fixed the Bot explorer and Resources panels to set the focus on the first interactive element

### Screenshots

Test cases executed successfully
resourcesBar component
imagen

botExplorerBar component
imagen

botNotOpenExplorer component
imagen

servicePane component
imagen

### CODE REVIEW

File: packages/sdk/ui-react/src/layout/expandCollapse/expandCollapse.tsx [hunk 1]
```diff
@@ -119,4 +121,11 @@ export class ExpandCollapse extends React.Component<ExpandCollapseProps, ExpandC
       this.onToggleExpandedButtonClick();
     }
   };
+
+  private setElementRef = (ref: HTMLElement): void => {
+    const { elementRef } = this.props;
+    if (elementRef) {
+      elementRef(ref);
```
Comment (tonyanziano) [inline packages/sdk/ui-react/src/layout/expandCollapse/expandCollapse.tsx:128]: I think renaming this `elementRef` prop to something like `elementRefHandler` would make this code easier to understand.

File: packages/app/client/src/ui/shell/explorer/servicePane/servicePane.tsx [hunk 2]
```diff
@@ -45,6 +45,7 @@ export interface ServicePaneProps extends ServicePaneState {
   title?: string;
   ariaLabel?: string;
   sortCriteria?: string;
+  elementRef?: (ref: HTMLElement) => void;
```
Comment (tonyanziano) [inline packages/app/client/src/ui/shell/explorer/servicePane/servicePane.tsx:48]: Same as below

File: packages/app/client/src/ui/shell/explorer/botNotOpenExplorer/botNotOpenExplorer.tsx [hunk 3]
```diff
@@ -40,6 +40,7 @@ export interface BotNotOpenExplorerProps {
   hasChat?: boolean;
   openBotFile?: () => Promise<any>;
   showCreateNewBotDialog?: () => void;
+  elementRef?: (ref: HTMLElement) => void;
```
Comment (tonyanziano) [inline packages/app/client/src/ui/shell/explorer/botNotOpenExplorer/botNotOpenExplorer.tsx:43]: Same as below

File: packages/sdk/ui-react/src/layout/expandCollapse/expandCollapse.tsx (hunk 1 above)
Comment (tonyanziano) [inline packages/sdk/ui-react/src/layout/expandCollapse/expandCollapse.tsx:128]: I think renaming this `elementRef` prop to something like `elementRefHandler` would make this code easier to understand.

**Review by tonyanziano — CHANGES_REQUESTED** [review]

File: packages/sdk/ui-react/src/layout/expandCollapse/expandCollapse.tsx (hunk 1 above)
Comment (luislera) [reply packages/sdk/ui-react/src/layout/expandCollapse/expandCollapse.tsx]: Done!

**Review by tonyanziano — APPROVED** [review]
Awesome, thank you!

**Comment by tonyanziano** [comment]
@luislera it seems like there are some merge conflicts that are preventing this from being merged

**Pull request merged**